feat: docs on AUTHORS/CONTRIBUTORS files (#326)
ansys/pyansys-dev-guide: doc/source/packaging/code/authors_code.rst

.. code:: text

    # This is the list of <PROJECT-NAME>'s significant contributors.
    #

>     # This file does not necessarily list everyone who has contributed code,

Reviewer: ```suggestion
    # This file does not necessarily list everyone who has contributed code.
```

    #

>     # For contributions made under a Corporate CLA, the organization will
>     # appear in this file.

Reviewer: ```suggestion
    # For contributions made under a Corporate CLA, the organization is
    # added to this file.
```
